riva/proto/riva_tts.proto

```diff
@@ -80,6 +80,9 @@ message SynthesizeSpeechRequest {
   // Zero Shot model params
   ZeroShotData zero_shot_data = 6;
 
+  // custom_dictionary parameters
```

Comment:
@manishaj-nv could you please update the description of the field for more clarity?